compose/compose-v2/index.md

```diff
@@ -1,45 +1,158 @@
 ---
 description: Key features and use cases of Docker Compose
 keywords: documentation, docs, docker, compose, orchestration, containers, uses, features
-title: Compose V2 Overview
+title: Evolution of Compose
+redirect_from:
+ - /compose/cli-command-compatibility/
 ---
 
-## Compose V2 and the new `docker compose` command
+This page provides information on the history of Compose and explains the key differences between Compose V1 and Compose V2. 
 
-> Important
->
-> The new Compose V2, which supports the `compose` command as part of the Docker
-> CLI, is now available.
->
-> Compose V2 integrates compose functions into the Docker platform, continuing
-> to support most of the previous `docker-compose` features and flags. You can
-> run Compose V2 by replacing the hyphen (`-`) with a space, using `docker compose`,
-> instead of `docker-compose`.
-{: .important}
+## History
+
+The first release of Compose, written in Python, happened at the end of 2014. Between 2014 and 2017, two more versions of Compose were released:
 
-If you rely on using Docker Compose as `docker-compose` (with a hyphen), you can
-set up Compose V2 to act as a drop-in replacement of the previous `docker-compose`.
-Refer to the [Installing Compose](../install/index.md) section for detailed instructions.
+- [Compose 1.6.0 with file format V2](../compose-file/compose-file-v2/)
+- [Compose 1.10.0 with file format V3](../compose-file/compose-file-v3/)
 
-## Context of Docker Compose evolution
+These three early releases are collectively referred to as Compose V1. 
```

Comment:
There was way more than 3 releases for Compose v1, the last one is the `v1.29.2` 😅 
But the `1.6.0` and `1.10.0` were noticeable as they introduced version `2` and `3` of the Compose file format.

Reply:
Ah that's actually what i was trying to communicate in that section - that there were 3 main releases within Compose V1 which are the file formats, not that there were only three releases 😅  But I've obviously not done a good job, so I'll try again! Thank you, Guillaume!

Reply:
Maybe we can reword this way:
```markdown
The first release of Compose, written in Python, happened at the end of 2014. Between 2014 and 2017, two other noticeable versions of Compose, as they introduced new Compose file format version, were released:
```